frontend/pulse3d/src/components/uploads/InteractiveAnalysisModal.js

```diff
@@ -117,6 +117,11 @@ const constantModalLabels = {
     messages: ["There was an issue while attempting to start this analysis.", "Please try again later."],
     buttons: ["Close"],
   },
+  duplicate: {
+    header: "Before Running Analysis!",
+    messages: ["Check no double peaks or valleys exist", "(Red Triangles)"],
```

Comment:
This might be better to say something like 'They are shown as red triangles above.' and  add a period at the end of the first line